### DISCUSSION TITLE
Installation of BSgenome.Hsapiens.UCSC.hg19 by default

### DISCUSSION THREAD

**Discussion by NiharikaCUSB** [gagneurlab/drop#522] [Q&A]
I have installed BSgenome.Hsapiens.UCSC.hg38 and want to incorporate this in drop_env_133.

**Comment by vyepez88** [comment]
We usually don't include the BSgenome in the conda env as it is too heavy and relatively easy for each user to install

**Comment by NiharikaCUSB** [comment]
Thank you for your prompt reply, since my input BAM files are aligned on genome release [GRCh38.p14] I wanted DROP to use the same version of genome. If I ignore installing the latest release will i be able to run DROP given my files are aligned on aforesaid release. 
I've tried running but I am getting an error [log attached] 
[2024-02-28T182112.885006.snakemake.log](https://github.com/gagneurlab/drop/files/14434383/2024-02-28T182112.885006.snakemake.log)
